connectors/es/sink.py

```diff
@@ -54,6 +54,7 @@
 OP_UPSERT = "update"
 OP_DELETE = "delete"
 TIMESTAMP_FIELD = "_timestamp"
+CANCELATION_TIMEOUT = 5
```

Comment:
This feels like a good candidate to add to service configuration in our YAML file, so that it can be easily increased if necessary.

Reply:
The reason why I didn't do so:

1. Although flexibility is always good, I don't feel customers want to configure this.
2. We currently pass `elastic_config` to `SyncOrchestrator`, and apparently this is not part of `elastic_config`, and I'm hesitate to introduce a new param to `SyncOrchestrator`:
https://github.com/elastic/connectors-python/blob/0c69b9f5cf87d004f5f67422fd7b57953a287daf/connectors/es/sink.py#L629-L640

Reply:
Ok, we can leave this for https://github.com/elastic/enterprise-search-team/issues/5757